src/main/java/com/friendly_machines/intellij/plugins/ideanative2debugger/impl/GdbOsProcessHandler.java

```diff
@@ -121,7 +125,12 @@ public void notifyTextAvailable(@NotNull String text, @NotNull Key outputType) {
             ApplicationManager.getApplication().invokeLater(() -> {
                 var debugProcess = (DebugProcess) GdbOsProcessHandler.this.getUserData(DebugProcess.DEBUG_PROCESS_KEY);
                 if (debugProcess != null) {
-                    debugProcess.processAsync(token, scanner);
+                    try {
+                        debugProcess.processAsync(token, scanner);
+                    } catch (IOException | InterruptedException e) {
+                        // pucgenie: So who catches this?
```

Comment:
Good question. This is running in IDEA's GUI thread, but we are using their invokeLater--so hopefully they will handle RuntimeExceptions already.

Maybe just test to throw one and see what happens.

Reply:
I tested it and I got a scary message that a plugin caused an IDEA error and must be reported.